src/app/app.module.ts

```diff
@@ -97,9 +95,18 @@ registerLocaleData(localeUk);
   bootstrap: [AppComponent]
 })
 export class AppModule {
-  constructor() {
-    localStorage.setItem('ui-culture', 'uk');
+  constructor(private dateAdapter: DateAdapter<Date>) {
+    dateAdapter.setLocale(selectedLanguage);
```

Comment:
Can we move this to app.component? 
It is better to avoid adding logic to the module.